GWAS/LD_Clumping.ipynb

```diff
@@ -205,6 +205,9 @@
     "parameter: clump_kb = 2000\n",
     "# For cluster jobs, number commands to run per job\n",
     "parameter: job_size = 1\n",
+    "# File names for clumping\n",
+    "parameter: clumpFile = path(f'{cwd}/' + \"_\".join([f'{x:bn}' for x in sumstatsFiles]) + '.clumped')\n",
+    "parameter: clumpregionFile = path(f'{cwd}/' + \"_\".join([f'{x:bn}' for x in sumstatsFiles]) + '.clumped_region')\n",
```

Comment:
@dianacornejo could you remove `clumpregionFile` parameter, and at the end of global section add:

```
clumpregionFile = f'{clumpFile:n}.clumped_region'
```

so users dont have to input both?